Source/plugins/IShell.h

```diff
@@ -235,6 +235,9 @@ namespace PluginHost {
         //! Substituted Config value
         virtual string Substitute(const string& input) const = 0;
 
+        //! Substituted environments list
```

Comment:
As described above, if we carry this information in the RPC::Object (Here: https://github.com/rdkcentral/Thunder/blob/master/Source/com/Communicator.h#L46 ) I guess there is no need to "substitute" here. It can be done here: https://github.com/rdkcentral/Thunder/blob/master/Source/Thunder/PluginServer.h#L1288 as the environment variables are now part of the RPC::Object And no need to

Reply:
done